Reservation/app/mongohelper.go

```diff
@@ -21,7 +21,7 @@ type MongoHelper struct {
 
 // Connect to the MongoDB
 func CreateMongoConnection() *MongoHelper {
-	uri := os.Getenv("MONGO_CONNECTIONSTRING")
+	uri := os.Getenv("mongo_connectionstring")
```

Comment:
>mongo_connectionstring [](start = 19, length = 22)

It might make sense to add a `const` section at the top of the file for these.  There should be examples of that in the other .go files

Reply:
Sure

---
In reply to: [266645796](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/20#discussion_r266645796) [](ancestors = 266645796)